src/api/LotsService.ts

```diff
@@ -15,8 +15,14 @@ class LotsService {
   getLotsData = async (): Promise<AxiosResponse<PartsInterface[]>> =>
     storeApi.get<PartsInterface[]>(`${this.server}:${this.port}/lots`);
 
+  createLot = async (payload: any): Promise<AxiosResponse<PartsInterface>> =>
```

Comment:
please replace `any` with corresponding type (around all document)

Reply:
I'll make it in next develop iteration